### DISCUSSION TITLE
TinyXML2 library

### DISCUSSION THREAD

**Discussion by scarponiv** [sofa-framework/sofa#4700] [Build / Config / Environment] [answered]
Hi everyone,

If it can be helpful to anyone, when updating SOFA to version 23.12, if you encounter the error `Could NOT find TinyXML2` during the cmake configuration, you can resolve it by installing TinyXML2 with the command `sudo apt-get install libtinyxml2-dev` on Linux.

Have a nice day! :)

**Comment by epernod** [comment, marked as answer]
Thanks for the hint @scarponiv .
This has also been added to the last documentation on how to install SOFA: https://www.sofa-framework.org/community/doc/getting-started/build/linux/#core-required
